### PR TITLE
Light mode optimizations.

### PR DESCRIPTION
**Pull request by samelhusseini**
- Disable transitions
- Pass light mode to simulator on initialization and through runoptions
- Disable loading and dimmer animations
- Remove blockly workspace and drag filters
- Remove simulator reveal and remove transitions

Still yet to come, dynamically enabling light mode when compile is slow.

### CODE REVIEW

File: pxtsim/simdriver.ts [hunk 1]
```diff
@@ -130,6 +131,7 @@ namespace pxsim {
             frame.setAttribute('sandbox', 'allow-same-origin allow-scripts');
             frame.sandbox.value = "allow-scripts allow-same-origin"
             let simUrl = this.options.simUrl || ((window as any).pxtConfig || {}).simUrl || "/sim/simulator.html"
+            if (light) simUrl+= '?light=1';
```
Comment (pelikhan) [inline pxtsim/simdriver.ts]: This breaks offline caching. Pass light in the simulator options

Comment (samelhusseini) [reply pxtsim/simdriver.ts]: I do do that, but issue is simulator options are only passed in once loaded. Which means by that time, the SVGs have been constructed. I needed a way to pass in that option on initialization. 
Thoughts?

Comment (pelikhan) [reply pxtsim/simdriver.ts]: You'll need to encode that info in the hashtag arguments

Comment (samelhusseini) [reply pxtsim/simdriver.ts]: Cool yeah that should be easy. thanks

File: webapp/public/custom.css [hunk 2]
```diff
@@ -220,4 +220,33 @@ div.blocklyWidgetDiv {
...
+    animation: none !important;
+}
+
+/* Remove blockly filter on drag */
+.light svg.blocklyBlockDragSurface > g {
+    filter: none !important;
+}
+
+.light .blocklyMainBackground {
+    fill: transparent !important;
```
Comment (pelikhan) [inline webapp/public/custom.css]: should we have a solid background to speed things up?

Comment (samelhusseini) [reply webapp/public/custom.css]: Yeah we could use the fill color for blocklySvg.

**Comment by pelikhan** [comment]
Other idea: increase debouncing time for autoRunBlocksSimulator / autoRunSimulator / typecheck

**Comment by pelikhan** [comment]
Let's make sure we track this. It is worth knowing if "light" mode is used a lot or not.

**Pull request merged**